Dockerfile

```diff
@@ -22,7 +22,6 @@ RUN echo " ... adding throttling support with ZMQ and CZMQ" \
          && cd /tmp/ \
          && tar -xf /tmp/zeromq.tar.gz \
          && cd /tmp/zeromq*/ \
-         && apk add automake autoconf \
```

Comment:
thanks @atrifan . IIRC these are only needed to build ZMQ. If that's the case we can only install them when compiling ZMQ libs.